tfhe/src/core_crypto/experimental/entities/mod.rs

```diff
@@ -0,0 +1,3 @@
+pub mod lwe_shrinking_keyswitch_key;
+
+pub use lwe_shrinking_keyswitch_key::*;
```

Comment:
IMO it's better to either only have `pub mod` or `mod and pub use` so that there is only one public path

Reply:
it's something we have in core crypto which allowed the user to either not care about the import and import all entities via a use * or use the full path

Reply:
I would prefer changing this after the release if we do, we have a lot of things to still complete and I don't see the value in doing that now vs after the release